sno/working_copy.py

```diff
@@ -286,11 +286,11 @@ def create(self):
 
             dbcur.execute(
                 f"""
-                CREATE TABLE {self.META_TABLE} (
+                CREATE TABLE {self.STATE_TABLE} (
                     table_name TEXT NOT NULL,
                     key TEXT NOT NULL,
                     value TEXT NULL,
-                    CONSTRAINT {self._meta_name('meta', 'pk')} PRIMARY KEY (table_name, key)
+                    CONSTRAINT {self._sno_table('meta', 'pk')} PRIMARY KEY (table_name, key)
```

Comment:
constraint name doesn't really _matter_ but we should keep it consistent with the table going forward, might as well change this to `state` too

Reply:
Done